docker/Dockerfile

```
@@ -23,28 +25,37 @@ RUN tar -C /usr/local -xzf go1.20.1.linux-amd64.tar.gz
 ENV PATH="/usr/local/go/bin:${PATH}"
 
 # Install TPM Build Dependencies
-RUN apt-get update && apt-get install -y libtss2-dev
+RUN apt-get install -y libtss2-dev
 
 # Install TDX Build Dependencies
 RUN curl -L https://download.01.org/intel-sgx/sgx_repo/ubuntu/intel-sgx-deb.key | apt-key add -
 RUN echo 'deb [arch=amd64] https://download.01.org/intel-sgx/sgx_repo/ubuntu focal main' | tee /etc/apt/sources.list.d/intel-sgx.list
 RUN apt-get update
-RUN apt-get install -y libtdx-attest-dev libsgx-dcap-quote-verify-dev
+RUN apt-get install -y \
+    libsgx-dcap-quote-verify-dev \
+    libtdx-attest-dev
 
 # Build and Install KBS
 ARG KBS_FEATURES=coco-as-builtin,rustls
 RUN cargo install --path src/kbs --no-default-features --features ${KBS_FEATURES}
 
 FROM debian:stable-slim
 
-RUN apt-get update && apt-get install apt-utils
-RUN apt-get install -y clang curl gnupg
+RUN apt-get update && apt-get install -y apt-utils
+RUN apt-get install -y \
+    clang \
+    curl \
+    gnupg
 
 # Install TDX Runtime Dependencies
 RUN curl -L https://download.01.org/intel-sgx/sgx_repo/ubuntu/intel-sgx-deb.key | apt-key add -
 RUN echo 'deb [arch=amd64] https://download.01.org/intel-sgx/sgx_repo/ubuntu focal main' | tee /etc/apt/sources.list.d/intel-sgx.list
 RUN apt-get update
-RUN apt-get install -y libtdx-attest libsgx-dcap-default-qpl libsgx-dcap-quote-verify tpm2-tools
+RUN apt-get install -y \
```

it might be leaner to also install without the recommended dependencies:
```suggestion
RUN apt-get install -y --no-install-recommends \
```